odd-platform-api/src/main/java/com/provectus/oddplatform/repository/DataEntityRepositoryImpl.java

```diff
@@ -182,6 +229,15 @@ public DataEntityRepositoryImpl(final DSLContext dslContext,
         return dtos;
     }
 
+    @Override
+    @Transactional
+    public void incrementViewCount(final DataEntityDetails dto) {
```

Comment:
Please don't use OpenAPI entities in repo layer. I suggest passing an ID here instead of an object

Reply:
Oops, I got lost in this abundance of DTOs. Fixed.